src/ConnectionManager.ts

```diff
@@ -125,6 +125,24 @@ export class ConnectionManager extends EventEmitter {
             this.push(res.connection);
             return res.connection;
         }
+        if (GitLabRepoConnection.EventTypes.includes(type)) {
+            if (!this.config.gitlab) {
+                throw Error('Generic hook support not supported');
```

Comment:
1. foo **support** is not **supported**
2. This is GitLab repos, not Generic hooks.

```suggestion
                throw Error('GitLab support not configured');
```

btw. the copy for Github just says "Github not configured" as if Microsoft had forgotten to configure it.

Reply:
Have tried to improve the wording for all.